<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerMainWindow</class>
 <widget class="QMainWindow" name="ServerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Le Carl Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="IP_label">
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IP_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="port_label">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_server_btn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_server_btn">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="the_graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>start_server_btn</tabstop>
  <tabstop>port_edit</tabstop>
  <tabstop>IP_edit</tabstop>
  <tabstop>the_graphicsView</tabstop>
  <tabstop>stop_server_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
